docs/_snippets/features/wproofreader.html

```diff
@@ -1,4 +1,7 @@
 <div id="snippet-wproofreader">
-	<p>Typos hapen. We striving to correct them. Hover on the marked words for instant correction suggestions or click the dialoge icon in the bottom right corner to have the whole text proofread at once.</p>
-	<p>You can also paste your own text here to have its spelling and grammar checked.</p>
+	<p>Warsaw is the capital adn largest city of Poland. The metropolis stands on the Vistula River in east-central Poland and its population is officially estimatd at 1.8 million residents within a greater metropolitan area of 3.1 million residents, which makes Warsaw the 7th most-populous capital city in the European Union. Warsaw is an alpha-global city, a major international tourist destination, and a significaant cultural, political, and economic hub. Its historical Old Town was designated a UNESCO World Heritage Site.</p>
```

Comment:
Is this the content written by yourself or borrowed from Wikipedia? If the latter, it would be good to reference the source.

Reply:
It is borrowed. I'll add this information at the bottom of the text in the snippet.